x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -308,6 +309,1604 @@ export const registerCollector: RegisterCollector = ({
               },
             },
           },
+          detection_rule_status: {
+            all_rules: {
+              eql: {
+                failed: {
+                  type: 'long',
+                  _meta: { description: 'The number of failed rules' },
+                },
+                top_failed: {
+                  type: 'array',
+                  items: {
+                    message: {
+                      type: 'keyword',
+                      _meta: { description: 'Failed rule message' },
+                    },
+                    count: {
+                      type: 'long',
+                      _meta: { description: 'Number of times the message occurred' },
+                    },
+                  },
+                },
+                partial_failure: {
+                  type: 'long',
+                  _meta: { description: 'The number of partial failure rules' },
+                },
+                top_partial_failure: {
```

Comment:
nit: maybe make plural? Is this the first top partial failure, or a list (seems it's an array) of top partials. If the latter would be more description to be plural for folks building dashboards.

Reply:
Sounds good. Will make this plural.